packages/component/src/BasicTranscript.tsx

```diff
@@ -731,7 +743,7 @@ const InternalTranscriptScrollable: FC<InternalTranscriptScrollableProps> = ({
         <section
           aria-roledescription={transcriptRoleDescription}
           className={classNames(activitiesStyleSet + '', 'webchat__basic-transcript__transcript')}
-          role="feed"
+          role={isWaiting ? undefined : 'feed'}
```

Comment:
Should we set `aria-busy` than modifying the `role`? I am worried by how screen reader behave when changing role.

Reply:
That's a good idea. Reading over what `aria-busy` does, this looks like it would make it do exactly what we want.